ldap_jwt_auth/routers/verify.py

```diff
@@ -0,0 +1,33 @@
+"""
+Module for providing an API router which defines a route for managing verification of JWT tokens.
+"""
+import logging
+from typing import Annotated
+
+from fastapi import APIRouter, Body, Depends, HTTPException, Response, status
+
+from ldap_jwt_auth.auth.jwt_handler import JWTHandler
+from ldap_jwt_auth.core.exceptions import InvalidJWTError
+
+logger = logging.getLogger()
+
+router = APIRouter(prefix="/verify", tags=["authentication"])
+
+
+@router.post(
+    path="/",
```

Comment:
I have just realised that these trailing `/`s are causing temporary redirects on SciGateway.

![image](https://github.com/ral-facilities/ldap-jwt-auth/assets/90245114/624ba787-f298-4285-9d76-1fe7f39477f8)

There doesn't seem to be an agreed convention on it but if we were using something like Django it would fail due to the difference. Probably best if they are removed here as that's what SciGateway expects (probably best to do it in the refresh PR in one go)

Reply:
Addressed this for login and verify endpoints as part of https://github.com/ral-facilities/ldap-jwt-auth/pull/9/commits/5989fe7eaa52e34975d3a66524ef87f93b424d96